src/wp-includes/class-wp-block-type.php

```diff
@@ -235,32 +235,32 @@ class WP_Block_Type {
-	 *     @type array|null    $attributes       Block type attributes property schemas.
-	 *     @type array         $uses_context     Context values inherited by blocks of this type.
-	 *     @type array|null    $provides_context Context provided by blocks of this type.
-	 *     @type string|null   $editor_script    Block type editor only script handle.
-	 *     @type string|null   $script           Block type front end and editor script handle.
-	 *     @type string|null   $view_script      Block type front end only script handle.
-	 *     @type string|null   $editor_style     Block type editor only style handle.
-	 *     @type string|null   $style            Block type front end and editor style handle.
+	 *     @type string            $api_version      Block API version.
+	 *     @type string            $title            Human-readable block type label.
+	 *     @type string|null       $category         Block type category classification, used in
+	 *                                               search interfaces to arrange block types by category.
+	 *     @type array|null        $parent           Setting parent lets a block require that it is only
+	 *                                               available when nested within the specified blocks.
+	 *     @type array|null        $ancestor         Setting ancestor makes a block available only inside the specified
+	 *                                               block types at any position of the ancestor's block subtree.
+	 *     @type string|null       $icon             Block type icon.
+	 *     @type string            $description      A detailed block type description.
+	 *     @type string[]          $keywords         Additional keywords to produce block type as
+	 *                                               result in search interfaces.
+	 *     @type string|null       $textdomain       The translation textdomain.
+	 *     @type array             $styles           Alternative block styles.
+	 *     @type array             $variations       Block variations.
+	 *     @type array|null        $supports         Supported features.
+	 *     @type array|null        $example          Structured data for the block preview.
+	 *     @type callable|null     $render_callback  Block type render callback.
+	 *     @type array|null        $attributes       Block type attributes property schemas.
+	 *     @type array             $uses_context     Context values inherited by blocks of this type.
+	 *     @type array|null        $provides_context Context provided by blocks of this type.
+	 *     @type string|null       $editor_script    Block type editor only script handle.
+	 *     @type string|null       $script           Block type front end and editor script handle.
+	 *     @type string|null       $view_script      Block type front end only script handle.
+	 *     @type array|string|null $editor_style     Block type editor only style handle.
```

Comment:
We should also document with `@since` that since 6.1.0 an array format is supported. We should also expand the description because it's no longer a style handle only.